packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -359,6 +359,26 @@ Expected response body:
 }
 ```
 
+### Azure_Core_Traits_create
+
+- Endpoint: `get /azure/core/traits`
```

Comment:
what are azure core traits?

Reply:
Hi @iscai-msft , this PR is trying to add test for all azure.core models.
I listed them in https://microsoftapc-my.sharepoint.com/:x:/g/personal/changll_microsoft_com/Ec6IfM_M-ItHgU13Y4oqlQEBNtNylhcCXstKCrsUbHwCEA?e=IOmwc5, this PR cover all `Traits` items in yellow color.